taxi/views.py

```diff
@@ -80,10 +83,44 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
 
 
 class DriverListView(LoginRequiredMixin, generic.ListView):
-    model = Driver
+    model = get_user_model()
     paginate_by = 5
 
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
-    model = Driver
-    queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+    model = get_user_model()
+    queryset = Driver.objects.prefetch_related("cars__manufacturer")
```

Comment:
Why use `get_user_model()` for all the `model` fields, but not for querysets? :)